galileo_sdk/galileo_cli/universes.py

```diff
@@ -29,7 +32,7 @@ def ls():
             ]
         ]
         spinner.stop()
-        click.echo(universes_df.head(1))
+        click.echo(universes_df.head(10))
```

Comment:
The description says this will list all universes, but this looks like it might be capping the list to 10

Reply:
yup, i thought I had fixed that actually

Reply:
good point